### PR TITLE
Sdk feature parity

### PR DESCRIPTION
**Pull request by ftcvlad**
### Description of Changes

Android. Up to AndroidSDK 8.5.0 including
1) notification composer enhancements (big text, sendWhen, sounds)
2) inbox deletion

iOS. Up to objcSDK 4.2.2 including
1) delivery tracking
2) inbox deletion
3) incrementing badges

### Breaking Changes

-   None

### Release Checklist

Prepare:

-   [x] Detail any breaking changes. Breaking changes require a new major version number
-   [x] Update type declarations in index.d.ts

Bump versions in:

-   [x] package.json
-   [x] plugin.xml
-   [x] src/ios/KumulosSDKPlugin.m
-   [x] src/android/.../KumulosInitProvider.java

Release:

-   [ ] Squash and merge to master
-   [ ] Delete branch once merged
-   [ ] Create tag from master matching chosen version
-   [ ] Fill out release notes
-   [ ] Run `npm publish`

### CODE REVIEW

File: src/android/KumulosSDKPlugin.java [hunk 1]
```diff
@@ -311,6 +315,30 @@ private void inAppPresentInboxMessage(JSONArray args, CallbackContext callbackCo
...
+            return;
+        }
+
+        List<InAppInboxItem> items = KumulosInApp.getInboxItems(this.cordova.getContext());
+        for (InAppInboxItem item : items) {
+            if (item.getId() == messageId) {
+                Boolean result = KumulosInApp.deleteMessageFromInbox(cordova.getContext(), item);
+
+                if (result) {
+                    callbackContext.success();
```
Comment (cgwyllie) [inline src/android/KumulosSDKPlugin.java:332]: Should break/return here in both cases?

File: src/ios/KumulosSDKPlugin.m [hunk 2]
```diff
@@ -237,6 +238,29 @@ -(void)inAppPresentInboxMessage:(CDVInvokedUrlCommand*)command {
...
+-(void)inAppDeleteMessageFromInbox:(CDVInvokedUrlCommand*)command {
+    NSNumber* messageId = command.arguments[0];
+
+    NSArray<KSInAppInboxItem*>* inboxItems = [KumulosInApp getInboxItems];
+    for (KSInAppInboxItem* msg in inboxItems) {
+        if ([msg.id isEqualToNumber:messageId]) {
+             BOOL result = [KumulosInApp deleteMessageFromInbox:msg];
+
+            if (result) {
+                [self.commandDelegate
```
Comment (cgwyllie) [inline src/ios/KumulosSDKPlugin.m:250]: Same logic RE break/return?

File: src/android/KumulosSDKPlugin.java (hunk 1 above)
Comment (ftcvlad) [reply src/android/KumulosSDKPlugin.java:332]: fixed

File: src/ios/KumulosSDKPlugin.m (hunk 2 above)
Comment (ftcvlad) [reply src/ios/KumulosSDKPlugin.m:250]: fixed

**Pull request merged**